bosh-stemcell/lib/bosh/stemcell/build_environment.rb

```diff
@@ -181,9 +181,12 @@ def exclude_exclusions
       when 'google'
         ' --tag ~exclude_on_google'
       else
-        ''
+        nil
       end,
-      ].join(' ').rstrip
+      unless operating_system.variant.nil?
+        " --tag #{operating_system.variant}"
+      end,
+      ].compact.join(' ').rstrip
```

Comment:
The `exclusions` for specs to run has also confused me a number of times, so I'm probably not fully understanding what's going on here. But are we creating an exclusion here, or an inclusion? All the other tags use a `~` to exclude certain specs, but here we're just doing `--tag variant` which will exclude everything except specs tagged with the variant?

I suppose that would be excluding everything EXCEPT the variant? So maybe still technically an "exclusion" of sorts.

Reply:
Was hoping this would allow running additional tests only when building a `fips` stemcell. But it does not seem to work like that according to the docs: https://relishapp.com/rspec/rspec-core/v/2-4/docs/command-line/tag-option. Will make it an exclusion filter so that at least when building a fips stemcell certain tests could be excluded when needed.